Atomic/scan.py

```diff
@@ -34,32 +36,62 @@ def get_scan_info(scanner, scan_type):
                     for x in i['scans']:
                         if x['name'] == scan_type:
                             return i['image_name'], x['args'], i.get('custom_args')
+
+        def set_default_scanner():
+            self.default_scanner = self.atomic_config['default_scanner']
+
         if self.args.debug:
             self.debug = True
 
         # Set debug bool
         self.set_debug()
 
+        set_default_scanner()
+
+        # If no scanners are defined
+        if len(self.scanners) < 1:
+            count = 1
+            raise ValueError("No scanners are configured for your system.")
+        # If a scanner is defined in parseargs
+        elif self.args.scanner is not None:
+            count = 3
+            self.scanner = self.args.scanner
+        # If only one scanner is defined and no default is defined
+        elif len(self.scanners) == 1 and self.default_scanner is None and self.args.scanner is None:
+            count = 4
+            self.scanner = self.scanners[0]['scanner_name']
+        # If more than one scanner is defined but no default
```

Comment:
In setup_default_scanner, you should just set this

if len(self.scanners) == 1:
     self.default_scanner = THISSCANNER>